test-utils/src/main/kotlin/com/google/devtools/ksp/processor/ConstructorDeclarationsProcessor.kt

```diff
@@ -24,9 +24,10 @@ import com.google.devtools.ksp.symbol.*
 
 class ConstructorDeclarationsProcessor : AbstractTestProcessor() {
     val visitor = ConstructorsVisitor()
+    lateinit var result: List<String>
 
     override fun toResult(): List<String> {
-        return visitor.toResult()
+        return result
```

Comment:
what's this change for?

Reply:
Some KtSymbols are not available after cache drop